cranelift/filetests/filetests/isa/s390x/atomic_load-little.clif

```diff
@@ -41,29 +41,29 @@ block0:
 ; check:  larl %r1, %sym + 0 ; lrv %r2, 0(%r1)
 ; nextln: br %r14
 
-function %atomic_load_i16(i64) -> i16 {
+function %atomic_load_i16(i64) -> i32 {
 block0(v0: i64):
-  v1 = atomic_load.i16 little v0
+  v1 = atomic_uload16.i32 little v0
   return v1
 }
 
 ; check:  lrvh %r2, 0(%r2)
```

Comment:
The test case is likewise wrong.  You can look at load-little.clif for a complete set of load test cases.  Those should be usable for atomic loads as well (aligned loads are atomic by default on the architecture).